### DISCUSSION TITLE
Issue with web3auth and Apple login in mainnet

### DISCUSSION THREAD

**Discussion by sebastianupr** [Web3Auth/Web3Auth#552] [Help]
Hi guys, I have an issue in custom auth with Apple and auth0 verifier, has occurs only in mainnet, I’ve tested on testnet, mainnet and cyan, but only occurs on mainnet network.

The error message says:
 "Could not get result from torus nodes 
 Verifier not supported. Check if you: 

      1. Are on the right network (Torus testnet/mainnet) 

      2. Have setup a verifier on dashboard.web3auth.io?"

I didn’t find any solution for this, in my dashboard I have register apple with auth0 verifier with respective configuration and requirements, has been registered on the 3 networks and only fails on mainnet network, but it works great with Google, and we had no issues.

My team has implemented this login with tKey sdk, but we have tried with web3auth core, and it has the same error.

Thank you very much, I am attentive for any answer.

<img width="718" alt="Screen Shot 2022-09-05 at 5 10 59 PM" src="https://user-images.githubusercontent.com/42834763/188512762-b404937d-336c-42dd-8a7d-128e4311f2d3.png">
<img width="718" alt="Screen Shot 2022-09-05 at 5 11 37 PM" src="https://user-images.githubusercontent.com/42834763/188512801-ac149b15-c864-488f-bb41-587bb1fc222c.png">

**Comment by yashovardhan** [comment]
Hey @sebastianupr 

Sorry for missing this before, can you share your verifier details so that we can verify what's happening there? Also please check in your code that you're connecting to mainnet only.

**Comment by himanshuchawla009** [comment]
Is your verifier setup was done for mainnet or cyan?

**Reply by shahbaz17** [reply]
Hey @sebastianupr 

Friendly nudge. Could you reply with the progress and the questions our team asked?
